benchmark/UnsortedAndSortedJoinBenchmark.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright 2015, University of Freiburg,
```

Comment:
Wrote a function like `makeBenchmark` in [a5d3b62](https://github.com/ad-freiburg/qlever/pull/860/commits/a5d3b62d8a70ef3f66fb8792a20571bf47509fd1), which had quite a few bugs/mistakes, which where mostly fixed in [5eae936](https://github.com/ad-freiburg/qlever/pull/860/commits/5eae936fc035c5745c8a36f8c7dbdfc43ca511b0).

Additionaly, I also wrote the benchmark for "No overlap, smaller table size doesn't change, bigger table size grows" in [5eae936](https://github.com/ad-freiburg/qlever/pull/860/commits/5eae936fc035c5745c8a36f8c7dbdfc43ca511b0).

Reply:
Added "No overlap, specific ratio between the tables, smaller table grows" in [57c5272](https://github.com/ad-freiburg/qlever/pull/860/commits/57c5272a982c928d7cbb28f1857fc3ac44a899b4).

@joka921 Could you please take a look, if I understood correctly, what type of benchmark tables you wanted? I'm also not sure about the values, I used.

Reply:
TODO The numbers for the benchmarks are to small. They should go to 200.000.000.000 and the changes should follow a $x^i$ formulae for their growth, with $x=2$, or $x=10$.

Reply:
Changed numbers a wanted in [096cc40](https://github.com/ad-freiburg/qlever/pull/860/commits/096cc40a986e213e174b65232b35f7e98511573f). Now testing, if the new benchmarks can be run, without the computer killing the process.

Reply:
The computer of the faculty kills the process, so that was a bust.

Reply:
Adjusted the sizes, so that it now runs through in [ceb5128](https://github.com/ad-freiburg/qlever/pull/860/commits/ceb512815745ca63b1598b42633a19fa1444cc41).